src/social-network-provider/twitter.com/utils/selector.ts

```diff
@@ -90,7 +90,10 @@ export const postsContentSelector = () =>
         ].join(),
     ).concat(
         querySelectorAll('[data-testid="tweet"] > div:last-child').map(
-            (x) => x.querySelector('[role="group"]')?.parentElement?.firstElementChild as HTMLDivElement | undefined,
+            (x) =>
+                x.querySelector('[role="group"]')?.parentElement?.querySelector('div[lang]') as
+                    | HTMLDivElement
+                    | undefined,
```

Comment:
```suggestion
            (x) => x.closest<HTMLDivElement>('[role="group"] div[lang]'),
```

see https://mdn.io/Element.prototype.closest

Reply:
no necessary